apps/andi/lib/andi_web/views/error_helpers.ex

```diff
@@ -69,7 +59,13 @@ defmodule AndiWeb.ErrorHelpers do
 
   defp interpret_error(error, field) do
     {_, opts} = error
-    updated_message = "Please enter a valid #{get_downcased_display_name(field)}."
+
+    updated_message =
+      case field do
+        field when field in [:sourceHeaders, :sourceQueryParams] -> "Please enter valid key(s)."
```

Comment:
It would be nice if these generic error helpers weren't coupled to fields that are very specific to the dataset editor, but it's not the end of the wordl < Ben

Reply:
I agree.  My thinking was that the next card with custom error messaging would be the one to have to move this into a more sophisticated 'lookup' function.